README.md

```diff
@@ -41,24 +72,38 @@ Below are commands used for various testing and development scenarios.
 
 Before building the project, make sure to set the correct environment using the `NODE_ENV` variable. The build scripts for different environments are:
 
-- `yarn run build:dev` for development environment.
-- `yarn run build:staging` for staging environment.
-- `yarn run build:prod` for production environment.
+```bash
+yarn run build:dev
+```
+
+```bash
```

Comment:
I don't think we will have a `staging` environment for this project, so we should probably remove this